browser/ui/toolbar/brave_app_menu_model.cc

```diff
@@ -35,9 +35,13 @@ void BraveAppMenuModel::InsertBraveMenuItems() {
         GetIndexOfCommandId(IDC_SHOW_DOWNLOADS),
         IDC_SHOW_BRAVE_REWARDS,
         IDS_SHOW_BRAVE_REWARDS);
+    InsertItemWithStringIdAt(
```

Comment:
This one should be probably protected by build flag check

Reply:
@NejcZdovc this check here should be sufficient: https://github.com/brave/brave-core/pull/2496/files#diff-48d8778906af24da93d2c7079c17ec40R89

Are you seeing a case where the above does not work properly?